lib/getSrcForJoin.js

```diff
@@ -9,17 +79,13 @@
  * @return {Array}                      List of node object consists of job name and id
  */
 const getSrcForJoin = (workflowGraph, config) => {
-    const jobs = new Set();
+    let jobs = new Set();
```

Comment:
How about combine with L88 and using `const`?

Reply:
that is fine.
I fixed.